src/modules/likes.js

```diff
@@ -0,0 +1,42 @@
+const likesUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/UeRVpttbuH5idOpNpTZz/likes';
+
+const likePost = (likesId) => {
+  const likesCount = document.getElementById(`likes-count-${likesId}`);
+  if (likesCount) {
+    const currentLikes = parseInt(likesCount.textContent, 10);
+    const newLikes = currentLikes + 1;
+    likesCount.textContent = `${newLikes} Likes`;
+  }
+  fetch(likesUrl, {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: likesId,
+    }),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+};
```

Comment:
- [Required] I think since this function sends some data to an `API` online and it might take some time, it would be better if you use `async` and `await` with the `fetch` method to send the data. The keyword `async` before a function makes the function return a promise and the `await` keyword makes the function pause the execution and wait for a resolved promise before it continues which in turn makes your function run synchronously without an error. Please consider this change and rectify it.